mpa/cls/trainer.py

```diff
@@ -14,19 +14,20 @@
 
 import mmcv
 from mmcv.parallel import MMDataParallel, MMDistributedDataParallel
-from mmcv.runner import DistSamplerSeedHook, build_optimizer, build_runner, HOOKS
+from mmcv.runner import DistSamplerSeedHook, Fp16OptimizerHook, build_optimizer, build_runner, HOOKS
 
 from mmcls import __version__
 from mmcls.datasets import build_dataset, build_dataloader
 from mmcls.models import build_classifier
 from mmcls.utils import collect_env
-from mmcls.core import (DistOptimizerHook, Fp16OptimizerHook)
+from mmcls.core import DistOptimizerHook
```

Comment:
`DistOptimizerHook` is not deprecated? It's just a question, but I think that should be considered on this PR.

Reply:
We still using this. Please look at 241 line.

Reply:
I intended whether latest or 0.15.0 official mmclassification uses this hook or not removed yet deprecated hooks. (I remember they remove deprecated things on 0.23.0)

Reply:
I checked that it is still using in 0.15.0v.